client/my-sites/checkout/checkout-thank-you/premium-plan-details.jsx

```diff
@@ -44,7 +44,7 @@ const PremiumPlanDetails = ( { selectedSite, sitePlans, selectedFeature, purchas
 			/>
 
 			<PurchaseDetail
-				icon={ <img alt="" src="/calypso/images/upgrades/advertising-removed.svg" /> }
+				icon={ <img alt="Advertising Removed" src="/calypso/images/upgrades/advertising-removed.svg" /> }
```

Comment:
This `alt` attribute should be localized. By the way, I don't know accessibility rules very well but shouldn't it be better describe the element itself, i.e. `Advertising Removed logo`?